akka-sample-cqrs-scala/src/test/scala/sample/cqrs/ProjectionSpec.scala

```diff
@@ -0,0 +1,86 @@
+package sample.cqrs
+
+import java.io.File
+
+import akka.actor.testkit.typed.scaladsl.ScalaTestWithActorTestKit
+import akka.actor.typed.eventstream.EventStream
+import akka.persistence.cassandra.testkit.CassandraLauncher
+import akka.projection.testkit.scaladsl.ProjectionTestKit
+import com.typesafe.config.ConfigFactory
+import org.apache.commons.io.FileUtils
+import org.scalatest.BeforeAndAfterAll
+import org.scalatest.wordspec.AnyWordSpecLike
+
+object ProjectionSpec {
+  def config =
+    ConfigFactory.parseString("""
+      akka.actor.provider=local
+      akka.persistence.cassandra {		
+         events-by-tag {		
+           eventual-consistency-delay = 200ms		
+         }		
+       		
+         query {		
+           refresh-interval = 500 ms		
+         }		
+       		
+         journal.keyspace-autocreate = on		
+         journal.tables-autocreate = on		
+         snapshot.keyspace-autocreate = on		
+         snapshot.tables-autocreate = on		
+       }		
+       datastax-java-driver {		
+         basic.contact-points = ["127.0.0.1:19042"]		
+         basic.load-balancing-policy.local-datacenter = "datacenter1"		
+       }
+    """).withFallback(ConfigFactory.load()) // re-use application.conf other settings
+}
+
+class ProjectionSpec
```

Comment:
could you add corresponding example to the java sample too?

Reply:
Yep, just wanted a 👍 on that I'm not using the testkit in the wrong way first.